README.md

```diff
@@ -1 +1,116 @@
-# node-test-helpers
\ No newline at end of file
+Jasmine-Node Helpers
+====================
+*Helping you test your [Node.js] applications, one helper at a time.*
+
+[![Build Status][travis-image]][Travis CI]
+[![Dependencies][dependencies-image]][Dependencies]
+[![Dev Dependencies][devdependencies-image]][Dev Dependencies]
+[![codecov.io][codecov-image]][Code Coverage]
+
+About
+-----
+This contains a series of [Jasmine]1.3 helpers when using [Jasmine-Node] to facilitate testing certain functionality more easily. These help to detect when a test fails but should have passed, turn middleware into a promise and makes it easier to use promises in tests. More explanation is below for each helper.
+
+*Note: If you are not using [Jasmine-Node] these helpers aren't going to help your testing suite very much.*
+
+How to Use
+----------
+Include the package in your `package.json` file.
+
+```
+npm install --save-dev node-test-helpers
+```
+
+Then you need to be able to include the helper in your testing directory or where you'd like to run tests.
+
+You can do this by creating a symbolic link to files contained. You'll want to link to the files and not the folder. [Jasmine] doesn't seem to pick up on the files if only the folder is linked. Below is an example of setting up a symbolic link. Make sure to include the folder in your `.gitignore` file so you don't commit the files as this folder will show up in your status.
+
+```
+// Linux/MacOS
+mkdir ./path/to/project/test-folder/jasmine-node-helpers
+
+ln -s /path/to/project/from/root/node_modules/jasmine-node-helpers/lib/* ./path/to/project/test-folder/jasmine-node-helpers/
+
+// Windows
+// If you can figure out how to make symlink work, good on you, make a [fork](CONTRIBUTING.md) and update those instructions please.
+```
+
+jasmine.fail
+------------
+This is most useful when testing a promise which you are wanting to check the test is getting to the reject successfully, but want to fail in case code changes which makes the promise resolve instead of reject. This will make the test fail so it can be caught and code can be diagnosed.
```

Comment:
I think I get your meaning with the first sentence, but it was a little hard to parse at first. "This is most useful when testing that a promise is getting to the reject as expected, and when you'd like the test to fail if the promise is resolved rather than rejected." may be a little bit more clear (assuming I'm understanding it).